packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -61,13 +61,9 @@ export class Image extends BaseComponent<IImageProps, IImageState> {
   }
 
   public componentWillReceiveProps(nextProps: IImageProps) {
+    // If the image is not loaded, recompute the cover style.
     if (this.state.loadState === ImageLoadState.loaded) {
```

Comment:
It looks like computeCoverStyle looks at imageFit, width, and height. So we should only re-evalutate if one of those changes and the image is loaded.